SUIApp/SUIApp/ContentView.swift

```diff
@@ -7,19 +7,93 @@
 
 import SwiftUI
 
+/// ContentView
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constats {
+        static let defaultText = "Дефолтный текст"
+        static let defaultTwoText = "Дефолтный текст"
+        static let showAlertText = "Показать Алерт"
+        static let showTwoAlertText = "Показать алерт с 2 кнопками"
+        static let notDefaultText = "Недефолтный текст"
+        static let helloWorldText = "Hello, World!"
+        static let showActionSheetText = "Показать ActionSheet"
+        static let showTwoActionSheetText = "Показать ActionSheet с кнопками"
+        static let tapMeText = "Нажми меня"
+        static let spacingNumber: CGFloat = 20
+    }
+    
+    // MARK: - Private property
+    
+    @State private var defaultText = Constats.defaultText
+    @State private var defaultTwoText = Constats.defaultTwoText
+    @State private var isShowAlertOne = false
+    @State private var isShowAlertTwo = false
+    @State private var isShowActionSheetOne = false
+    @State private var isShowActionSheetTwo = false
+
+    
+    // MARK: - Public Property
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        VStack(spacing: Constats.spacingNumber) {
+            setupOneAlert()
+            setupTwoAlert()
+            setupOneActionSheet()
+            setupTwoActionSheet()
+        }
+        .font(.largeTitle)
+    }
+    
+    // MARK: - Private Methods
+    
+   private func setupOneAlert() -> some View {
```

Comment:
```suggestion
   private var oneAlertView: View {
```
Если нет параметров создаем не метод а вычисляемое, сделай ниже везде так